### PR TITLE
Added Hambug-menu

### PR DESCRIPTION
**Pull request by arnoldnekemiah**
Added the required changes
1. The mobile menu appears when the user clicks (or taps) the hamburger button.
2. The mobile menu disappears when the user clicks (or taps) the close (X) button.
3. The mobile menu disappears when the user clicks (or taps) any of the mobile menu options.
4. When the user clicks (or taps) any of the mobile menu options, the correct part of the page is displayed.

### CODE REVIEW

**Comment by arnoldnekemiah** [comment]
I implemented the following;

-     When the user clicks (or taps) the hamburger button, the mobile menu appears.
-     When the user clicks (or taps) the close (X) button, the mobile menu disappears.
-     When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
-     When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

I have done everything I can. I am still an amateur at DOM manipulation and so is my coding partner. The .specific-link content is not showing in the mobile view but when I try it out in the desktop view they show but not in the mobile view.
If you have a suggestion of how I can fix that I will appreciate so much as am behind schedule.

**Review by shella12 — COMMENTED** [review]
Hi @arnoldnekemiah , 

#  Status: Invalidated :red_circle: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that 

- [ ] Does not have contribution from partner. This milestone is a [pair-programming activity](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md#work-and-submission-mode). Please make sure you ask your part to contribute to your code before submitting another PR

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by arnoldnekemiah** [comment]
Collaborated with coding partner to make the following changes as required

-     The mobile menu appears when the user clicks (or taps) the hamburger button.
-     The mobile menu disappears when the user clicks (or taps) the close (X) button.
-     The mobile menu disappears when the user clicks (or taps) any of the mobile menu options.
-     When the user clicks (or taps) any of the mobile menu options, the correct part of the page is displayed.

**Comment by prolajumokeoni** [comment]
Hi @arnoldnekemiah 


While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request
You have submitted a pair-programming project, that does not have commit from your partner, and  that does not have the javascript linters also setup, for this project both the [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters should be configured and  kindly check the work[ mode submission](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md#work-and-submission-mode) requirements 👍🏿  
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by arnoldnekemiah** [comment]
Hello @prolajumokeoni , my coding partner has commits on this project. May be I forgot to run eslinters, that I will do But my partner made changes which I built on. I will checkout what you have suggested. Thank you

**Comment by arnoldnekemiah** [comment]
Collaborated with coding partner to make the following changes as required

-     The mobile menu appears when the user clicks (or taps) the hamburger button.
-     The mobile menu disappears when the user clicks (or taps) the close (X) button.
-     The mobile menu disappears when the user clicks (or taps) any of the mobile menu options.
-     When the user clicks (or taps) any of the mobile menu options, the correct part of the page is displayed.

My pair programming partner has committed to this project as required.

File: test.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    position: absolute;
+    z-index: 9999;
+    top: 0;
+    left: 0;
+    background-color: #3c3a39;
+    backdrop-filter: blur(6px);
+    width: 100%;
+    height: 100%;
+    display: flex;
+  }
```
Comment (MahmoudBakr23) [inline test.css]: - [ ] Please make sure to add the required styles that match the original design. Please look at the screenshot below of your website and compare it with the original Figma design and apply the necessary changes for the colors, font weight, size, and spacing (margin and padding).
Screenshot (1)

File: test.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+document.getElementById('btn-menu').addEventListener('click', toggleMenuPopup)
+const ad = document.getElementById('links')
+const closeBtn = document.querySelector('.close-popup');
+
+function toggleMenuPopup() {
+  ad.classList.toggle('menu-popup');
+}
+
+closeBtn.addEventListener('click', () => {
```
Comment (MahmoudBakr23) [inline test.js:7]: - [ ] Kindly add the necessary JS logic for the menu to totally disappear after clicking one of its nav links. See the screenshot below after an attempt to click on the contacts link.
Screenshot (2)

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello team😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Well done adding the nav menu for the mobile screen so far
- Descriptive PR and README file
- Extras for the live demo link and the loom video
- Correct Linters

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

- Please update your Live demo link to the latest changes of your website.

Cheers and Happy coding!👏👨‍💻💻

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by arnoldnekemiah** [comment]
- Updated Loom video
- - added the necessary JS logic for the menu to totally disappear after clicking one of its nav links.
- made the popup menu to resemble as much as possible the figma design

File: test.js [hunk 3]
```diff
@@ -1,14 +1,20 @@
...
   ad.classList.toggle('menu-popup');
 }
+document.getElementById('btn-menu').addEventListener('click', toggleMenuPopup);
+
+const closeBtn = document.querySelector('.close-popup');
 
 closeBtn.addEventListener('click', () => {
   ad.classList.remove('menu-popup');
-  document.body.classList.closeBtn('menu-open');
+  // document.body.classList.closeBtn('menu-open');
```
Comment (rustam-tolipov) [inline test.js]: - [ ] While you did the required changes by the previous reviewer, you left some places that need to be cleaned or fixed. Kindly remove unused commented codes from your project.

File: test.css [hunk 4]
```diff
@@ -6,26 +6,44 @@
 
   .specific-links {
     display: none;
+    /* width: 327px;
+    height: 56px; */
```
Comment (rustam-tolipov) [inline test.css]: - [ ] Please remove unused commented code to keep your project professional.

File: test.css [hunk 5]
```diff
@@ -6,26 +6,44 @@
...
+    width: 327px;
+    height: 184px;
+    left: 24px;
+    top: 92px;
+    gap: 8px;
```
Comment (rustam-tolipov) [inline test.css:13]: - [ ] Also, as the previous reviewer suggested. Kindly follow the Figma design that is given. You have some places in your Popup menu that need to be fixed to match the design. Your design should match the given design. Please fix the design as well. Thank you!

Your image            |  Original image
:-------------------------:|:-------------------------:
Screenshot 2023-01-03 at 18 29 21 |

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hi Team, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


 ## Highlights
Great readme file 💯 
Clean design 💯 
Kudos for adding video explanation 💯 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by arnoldnekemiah** [comment]
- Made final changes to match the figma design as suggested by the previous reviewer.
- Removed commented-out code as suggested by the reviewer.

**Review by mikethreels — APPROVED** [review]
Hi Arnold, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- linters are passing
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**